.github/workflows/main.yml

```diff
@@ -434,7 +434,7 @@ jobs:
         pacboy: binutils
       if: matrix.target == 'x86_64-pc-windows-gnu'
     - shell: p2sh
-      run: "C:\msys64\mingw64\bin" >> $env:GITHUB_PATH
+      run: "C:\\msys64\\mingw64\bin" >> $env:GITHUB_PATH
```

Comment:
```suggestion
      run: "C:\\msys64\\mingw64\\bin" >> $env:GITHUB_PATH
```
Surely?

Reply:
Yep. This is a WIP and not something I mean to submit as ready. I'm just forced to debug via my fork's GH CI, and every update I push to it is being mirrored here.

If the notifications are bothering you, I'll do my further tests on a branch. This was opened prematurely and should at least be marked a draft...

Reply:
No worries, I was just going through my notifications and happened upon this.